Ginger/Ginger/Drivers/DriversWindows/MobileDriverWindow.xaml.cs

```diff
@@ -647,22 +1093,26 @@ public void InitWindowLook()
         {
             try
             {
+                ImageMakerControl image = new ImageMakerControl();
                 //General
                 if (mDriver.GetDevicePlatformType() == eDevicePlatformType.Android)
                 {
-                    this.Icon = ImageMakerControl.GetImageSource(eImageType.AndroidOutline);
+                    this.Icon = ImageMakerControl.GetImageSource(eImageType.Android);
                 }
                 else
                 {
-                    this.Icon = ImageMakerControl.GetImageSource(eImageType.IosOutline);
+                    this.Icon = ImageMakerControl.GetImageSource(eImageType.Ios);
                 }
-                this.Title = string.Format("Ginger {0} Device View", mAgent.Name);
+
                 this.Width = 320;
                 this.Height = 650;
                 xMessageLbl.Content = "Connecting to Device...";
 
                 //Configurations
                 SetConfigurationsPanelView(false);
```

Comment:
why it is needed on init? don't we need only the title setup? rest grids view and data can be set only on metrics section expanding